### PR TITLE
Avoid double slash in paths.

### LINKED ISSUE #1208: Normalize paths during websocket URL construction
One of our clients reported having problems with using their proxy setup because the path CTL generated for WS URLs was not normalized (double `/`)

### PR DESCRIPTION
**Pull request by zmrocze**
Closes #1208 .

Pr doesn't do full path normalization, only avoids double slash "/" by replacing all snippets `<> "/" <>` with  "<</>>" operator.

### Pre-review checklist

- [X] All code has been formatted using our config (`make format`)
- [ ] The changelog has been updated under the `## Unreleased` header, using the appropriate sub-headings (`### Added`, `### Removed`, `### Fixed`), and the links to the appropriate issues/PRs have been included
  * Should this be included or to small of a change?

### CODE REVIEW

File: src/Internal/QueryM/ServerConfig.purs [hunk 1]
```diff
@@ -66,4 +67,4 @@ mkServerUrl protocol cfg =
     <> cfg.host
     <> ":"
     <> UInt.toString cfg.port
-    <> maybe "" ("/" <> _) cfg.path
+    <> maybe "" ("" <</>> _) cfg.path
```
Comment (klntsky) [inline src/Internal/QueryM/ServerConfig.purs:70]:
```suggestion
    <</>> fromMaybe "" cfg.path
```

**Review by klntsky — CHANGES_REQUESTED** [review]

**Comment by klntsky** [comment]
> Pr doesn't do full path normalization, only avoids double slash "/" by replacing all snippets <> "/" <> with "<</>>" operator.

This is OK

**Comment by klntsky** [comment]
@zmrocze please fix the compile-time warning (it's an error in CI). Otherwise good.

**Comment by zmrocze** [comment]
At last commit CI fails one test. I've just run it and passed. Before I also had cases of single tests (various ones) in Test.Ctl.Plutip test suite randomly failing. I think its some nondeterminism from plutip. 

Maybe if I now merge newest changes we get lucky run and all passes.

**Review by klntsky — APPROVED** [review]

**Pull request merged**